apcd-cms/src/apps/utils/apcd_database.py

```diff
@@ -1104,6 +1104,73 @@ def create_extension(form, iteration, sub_data):
         if conn is not None:
             conn.close()
 
+def update_extension(form):
+    cur = None
+    conn = None
+    try:
+        conn = psycopg2.connect(
+            host=APCD_DB['host'],
+            dbname=APCD_DB['database'],
+            user=APCD_DB['user'],
+            password=APCD_DB['password'],
+            port=APCD_DB['port'],
+            sslmode='require'
+        )
+        cur = conn.cursor()
+        operation = """UPDATE extensions
+            SET
+            updated_at= %s,"""
+        
+        set_values = []
+        # to set column names for query to the correct DB name
+        columns = {
+            'applicable-data-period': 'applicable_data_period',
```

Comment:
Optional: Again not related to this PR, just suggestions for a future task. 
1) The db column names could be const which enables differentiating which string literals are column names vs form names.
2) Also, a static map of list of form names to field name if it is possible.

Reply:
This is a great idea. I've added on this task to our portal work [https://jira.tacc.utexas.edu/browse/WP-185](https://jira.tacc.utexas.edu/browse/WP-185)